src/cameraManager.ts

```
@@ -0,0 +1,48 @@
+class MarsCameraManager {
+    private static instance: MarsCameraManager;
+    private cameras = new Map<CameraAbbreviation, { name: string; rovers: string[] }>();
+
+    private constructor() {
+        Object.entries(cameraData).forEach(([key, value]) => {
+            this.cameras.set(key as CameraAbbreviation, value);
+        });
+    }
+
+    public static getInstance(): MarsCameraManager {
+        if (!this.instance) {
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)